config/example.json

```diff
@@ -3,7 +3,8 @@
     "deployment": "openwhisk",
     "update_code": false,
     "update_storage": false,
-    "download_results": false,
+    "download_results": false, 
+    "architecture": "x86_64", 
```

Comment:
I think we should also add to `config/systems.json` a list of supported architectures. It will be x86 only for everything except for AWS. 

We can use it to fail early in case of wrong input, e.g., a user trying to run ARM functions on Azure.

Reply:
Yes, I updated it. 
Added in the systems.json for which platform what architecture are supported and fail early in case of the platform does not support the architecture as specified by the user